crates/iota-network/build.rs

```diff
@@ -12,6 +12,9 @@ use tonic_build::manual::{Builder, Method, Service};
 type Result<T> = ::std::result::Result<T, Box<dyn std::error::Error + Send + Sync>>;
 
 fn main() -> Result<()> {
+    println!("cargo::rustc-check-cfg=cfg(msim)");
+    println!("cargo::rustc-check-cfg=cfg(fail_points)");
```

Comment:
For what are these printed here?

Reply:
This crate's `build.rs` file seems to override the configuration from the main `Cargo.toml` so this is doing it in there.

Reply:
How does just printing it out change the configuration?

Reply:
Ah because it's build.rs

Reply:
The output of the `build.rs` file becomes the config for the crate compilation.

> Build scripts communicate with Cargo by printing to stdout. Cargo will interpret each line that starts with cargo:: as an instruction that will influence compilation of the package. All other lines are ignored.